src/common/uri.ts

```diff
@@ -101,3 +101,20 @@ export function fromFileChangeNodeUri(uri: Uri): FileChangeNodeUriParams {
 		return null;
 	}
 }
+
+export function toInMemUri(uri: Uri, pullRequestModel: IPullRequestModel, commit: string, fileName: string, base: boolean): Uri {
```

Comment:
Looks like `toPRUri` is no longer being used and could be removed

Reply:
also I think it would be good to rename `fromPRUri` and update the return type for clarity